pkg/pulumiyaml/ast/template.go

```diff
@@ -694,7 +699,15 @@ func (d *TemplateDecl) GetConfig() ConfigMapDecl {
 	if d == nil {
 		return ConfigMapDecl{}
 	}
-	// TODO: merge config and configuration (?)
+
```

Comment:
We change `GetConfig` to return `Config` rather than `Configuration`, and then have to update all the references